docs/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+**What is the problem / what does the code in this PR do**
+
+**Can you briefly describe how it works?**
+
+**Can you give an minimum working example (or illustrate with a figure)?**
```

Comment:
Sorry, but `an ...` -> `a minimum` and I think it is actually minimal working example.